deployment/CP05MOAS-A6264_Deploy.csv

```diff
@@ -1,99 +1,99 @@
+AR34,,AR34,,CP05MOAS-A6264-03-CTDAVN000,9,1,2019-04-05T22:12:00,2019-04-07T02:14:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.92028,-70.9071,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-04-NUTNRN000,9,1,2019-04-05T22:12:00,2019-04-07T02:14:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,39.92028,-70.9071,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-05-ADCPAN000,9,1,2019-04-05T22:12:00,2019-04-07T02:14:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.92028,-70.9071,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-06-PARADN000,9,1,2019-04-05T22:12:00,2019-04-07T02:14:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.92028,-70.9071,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-00-ENG000000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.91693,-70.906,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-01-FLORTN000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01646,39.91693,-70.906,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-02-DOSTAN000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,39.91693,-70.906,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-03-CTDAVN000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.91693,-70.906,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-04-NUTNRN000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,39.91693,-70.906,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-05-ADCPAN000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.91693,-70.906,,,,across-shelf survey
+AR34,,AR34,,CP05MOAS-A6264-06-PARADN000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.91693,-70.906,,,,across-shelf survey
 WJ20190617,,WJ20190617,,CP05MOAS-A6264-00-ENG000000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
 WJ20190617,,WJ20190617,,CP05MOAS-A6264-01-FLORTN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
-WJ20190617,,WJ20190617,,CP05MOAS-A6264-02-DOSTAN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
+WJ20190617,,WJ20190617,,CP05MOAS-A6264-02-DOSTAN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)

```

Comment:
Looks good. Associated checkout document .zip file will need to be revised - remove the 1520 cal doc and insert the 1519 cal doc. (3403-40001-00018-A.zip)